api/src/test/kotlin/org/jetbrains/kotlinx/dl/api/core/activation/ActivationTest.kt

```diff
@@ -31,4 +31,16 @@ open class ActivationTest {
             )
         }
     }
+
+    protected fun Activation.apply(input: FloatArray): FloatArray = EagerSession.create().use { session ->
```

Comment:
Split `assertActivationFunction` into 2 blocks for easier composition (in case someone wants to assert results differently):
1. apply activation to input and produce output
2. actually assert result

WDYT?

Reply:
it's an interesting and useful idea (with emulation of apply function on EagerSession). Hope it will be widely used in tests for primitives: initializers, layers, and so on.